### PR TITLE
Navigation tab bar, improved styling, and navigate between settings and home

### PR DESCRIPTION
**Pull request by maryjabro**
Things changed:

- Navigation tab bar at the bottom. 
- You can go back and forth between home and settings.
- Improved Styling on Home Page to match theme
- Changed export function Calendar to CalendarScreen() since the name didnt match up in app.js and calendar.js

The video below does a full demo of our app. Im going to work on adding icons and tweaking styling tomorrow. 

https://github.com/user-attachments/assets/328566ec-f0aa-4633-916d-e97404eab263

### CODE REVIEW

File: VitalTrackApp/src/screens/Settings.js [hunk 1]
```diff
@@ -1,17 +1,27 @@
...
 const SettingsScreen = () => {
+  const navigation = useNavigation();
+
   return (
     <View style={styles.container}>
       <Text style={styles.title}>Settings</Text>
 
+      <TouchableOpacity style={styles.backButton}
+        onPress={() => navigation.navigate('Home')}>
+        <Text style={styles.backButtonText}>Back to Home</Text>
```
Comment (MerwinAamir1) [inline VitalTrackApp/src/screens/Settings.js:22]: Do we need this? I believe theres a built in back button. In my opinon this just looks weird.

Comment (maryjabro) [reply VitalTrackApp/src/screens/Settings.js:22]: If there's a built in back button we dont need it. However, on my end there wasn't one so I was getting stuck on the settings page. Ill double check to see what went wrong

Comment (maryjabro) [reply VitalTrackApp/src/screens/Settings.js]: checkout update: 

https://github.com/user-attachments/assets/9246b5d6-704a-43a6-94a2-4393417db0a6

Comment (kparajuli21) [reply VitalTrackApp/src/screens/Settings.js]: It looks good!

**Review by kparajuli21 — APPROVED** [review]

**Pull request merged**